configure.ac

```diff
@@ -94,10 +97,12 @@ else
                 PLUGIN_LDFLAGS="-export-dynamic -avoid-version"
                 DEBUG="-g -Wall -D__NO_MATH_INLINES -fsigned-char"
                 CFLAGS="-O20 -D__NO_MATH_INLINES -fsigned-char"
+                CPPFLAGS="-O20 -D__NO_MATH_INLINES -fsigned-char"
```

Comment:
CPPFLAGS are for the preprocessor, so maybe move the defines to just CPPFLAGS I think, or use CXXFLAGS instead. (Same below)